docs/source/contrib.rst

```diff
@@ -74,6 +74,51 @@ The following process ensures correct version management:
  - Update the README file and add an empty placeholder for unreleased changes.
  - Merge the branch
 
+Updating pydoctor in Debian and Ubuntu
+--------------------------------------
+
+There shouldn't be any additional steps needed to get pydoctor updated in
```

Comment:
Maybe we can keep it generic... with a specific instruction for Debian.

```suggestion
Updating pydoctor for Linux distributions
-----------------------------------------

The information below covers Debian and its derivative distributions.
The same principles should be applied for Fedora, Arch, Alpine or any other Linux distribution.
 
There shouldn't be any additional steps needed to get pydoctor updated in
```

Reply:
Yes, being more generic is probably better, I was just looking through my usual Debian glasses. :smiley: 
Happily also take this suggestion.